ow_lander/src/ow_lander/server.py

```diff
@@ -122,6 +122,19 @@ def _set_aborted(self, msg, **kwargs):
     if self.fault_handler:
       self.fault_handler.notify_aborted()
 
+  def __should_reject_due_to_fault(self):
+    """Return true if the current state of faults and simulation settings permit
+    an action to run.
+    """
+    return (
+      # Actions may proceed if the simulation is configured such that goal
+      # errors are automatically cleared by a subsequent successful run.
+      not rospy.get_param("/lander_action_servers/auto_clear_goal_errors", True)
```

Comment:
Given this is a parameter, can one dynamically change its value using dynamic_reconfigure?

Reply:
No. It does not show up in the dynamic reconfigure GUI. I had not considered doing it that way, but it could be. If it becomes a dynamic reconfigure value, I certain if it can also be settable as a launch file argument.

What would be more preferable: setting it from command line, or setting it from rqt? If I can do both I will.

Reply:
Ideally both.  If that's hard, I'd prefer RQT, but maybe only if the default could still be true (auto-clear).  A command-line option alone is acceptable as is, it's just more cumbersome.

Reply:
Actually I got an easy solution that only took a couple lines of changes. It's a little hacky but it should do the trick. Update with it later tonight.

Reply:
I'm done with the given tests (plus a few of my own) and code review, and would approve this PR as is.  But I'd like to try the dynamic reconfigure option, so I'll wait for that...